src/grt/src/fastroute/src/FastRoute.cpp

```diff
@@ -721,12 +721,12 @@ NetRouteMap FastRouteCore::getPlanarRoutes()
         // defines the layer used for horizontal edges are still 2D
         int layer_v = 0;
 
-        if (layer_orientation_ != 0) {
-          layer_h = 1;
-          layer_v = 2;
+        if (((nets_[netID]->getMinLayer() % 2) - layer_orientation_) != 0) {
```

Comment:
Can you remind me what is this function used for? Why layers 1 and 2 were the default before?

Reply:
This is checking the orientation of the wires on a certain layer. In this function we are converting the nets routing from fastroute to segments so the MakeWireParasitics can understand them.
It was defaulted as 1 and 2 before because they were the first routing layers for the designs, but I changed it because it was an assumption, and some nets might have the minimal layer changed.

Reply:
Thanks! Your change looks good and makes sense with your new modeling.